docs/user/README.md

```diff
@@ -0,0 +1,13 @@
+# Docker Registry Module
+
+## What is Docker Registry?
+
+<!-- TODO: add more information -->
+
+## What is Docker Registry in Kyma?
+
+<!-- TODO: add more information -->
```

Comment:
Docker Registry brings extra capability to kyma runtime, useful in the development phase. It allows developers to push their images into kyma's internal registry and use the images to spin workloads in the same cluster.